src/Network/Network/help/New-AzBastionShareableLink.md

```diff
@@ -0,0 +1,256 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version: https://learn.microsoft.com/powershell/module/az.network/new-azbastionshareablelink
+schema: 2.0.0
+---
+
+# New-AzBastionShareableLink
+
+## SYNOPSIS
+The Bastion Shareable Link feature lets users connect to a target resource (virtual machine or virtual machine scale set) using Azure Bastion without accessing the Azure portal.
+
+## SYNTAX
+
+### ByResourceGroupNameByName (Default)
+```
+New-AzBastionShareableLink -ResourceGroupName <String> -Name <String>
+ -TargetVmId <System.Collections.Generic.List`1[System.String]> [-AsJob]
+ [-DefaultProfile <IAzureContextContainer>] [-ProgressAction <ActionPreference>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### ByResourceId
+```
+New-AzBastionShareableLink -ResourceId <String> -TargetVmId <System.Collections.Generic.List`1[System.String]>
+ [-AsJob] [-DefaultProfile <IAzureContextContainer>] [-ProgressAction <ActionPreference>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### ByInputObject
+```
+New-AzBastionShareableLink -InputObject <PSBastion>
+ -TargetVmId <System.Collections.Generic.List`1[System.String]> [-AsJob]
+ [-DefaultProfile <IAzureContextContainer>] [-ProgressAction <ActionPreference>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+Creates a Bastion Shareable Links for all the VMs specified in the request.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> $vm = Get-AzVM -ResourceGroupName $RgName -Name $vmName
+PS C:\> New-AzBastionShareableLink -ResourceGroupName $RgName -Name $bastionName -TargetVmId $vm.Id
+```
+
+```output
+{
+  "vm": {
+    "id": "/subscriptions/subid/resourceGroups/rgx/providers/Microsoft.Compute/virtualMachines/vm1"
+  },
+  "bsl": "http://bst-bastionhostid.bastion.com/api/shareable-url/tokenvm1",
+  "createdAt": "2019-10-18T12:00:00.0000Z"
+}
+```
+
+### Example 2
+```powershell
+PS C:\> $vm1 = Get-AzVM -ResourceGroupName $RgName -Name $vmName1
+PS C:\> $vm2 = Get-AzVM -ResourceGroupName $RgName -Name $vmName2
+PS C:\> $bastion = Get-AzBastion -ResourceGroupName $RgName -Name $bastionName
+PS C:\> New-AzBastionShareableLink -InputObject $bastion -TargetVmId $vm1.Id, $vm2.Id
```

Comment:
wonder if there can be space between target vm ids

Reply:
Should not block